inference-engine/tools/vpu/vpu_compile/main.cpp

```diff
@@ -25,6 +25,11 @@ static constexpr char model_message[] = "Required. Path to xml model.";
 static constexpr char plugin_path_message[] = "Optional. Path to a plugin folder.";
 static constexpr char output_message[] = "Optional. Path to the output file. Default value: \"<model_xml_file>.blob\".";
 static constexpr char config_message[] = "Optional. Path to the configuration file. Default value: \"config\".";
+static constexpr char platform_message[] = "Optional. Specifies movidius platform."
```

Comment:
Please, mention that PLATFORM config keys and values are deprecated and may not be supported at next releases